### DISCUSSION TITLE
Will backward compatibility be added/maintained in future releases?

### DISCUSSION THREAD

**Discussion by Nemgathos** [MeViMo/youbit#13] [Q&A] [answered]
I just stumbled upon [`YouBit`’s release page](https://github.com/MeViMo/youbit/releases) where in section `v2.0.0` it says:

> **Not backward compatible with existing YouBit videos made with YouBit < v2.0.0**

Is it possible to add backward compatibility for these videos in future releases? Right now, there is an [example video](https://www.youtube.com/watch?v=dnhlx48t-h4) referenced in the [`README.md` file](https://github.com/MeViMo/youbit/blob/main/README.md) that does not work using the current `v2.0.0` release.

**Comment by flonle** [comment, marked as answer]
The example video was transitioned to YouBit v2 and should work. If it does not, something else is the cause. I haven't tested that video since the v2 release.

EDIT: I just tested the example video and I could decode it succesfully.

**Reply by Nemgathos** [reply]
I used an [older version](https://github.com/MeViMo/youbit/discussions/11#discussioncomment-4408660).

**Comment by flonle** [comment]
Old uploaded videos should be either decoded with the appropriate version of YouBit, or have their description altered so that it is recognized by the newer version, although the latter may prove tricky.
